lib/medical_records/client.rb

```diff
@@ -13,7 +13,7 @@ class Client < Common::Client::Base
     include Common::Client::Concerns::MHVJwtSessionClient
 
     # Default number of records to request per call when searching
-    DEFAULT_COUNT = 100
+    DEFAULT_COUNT = 9999
```

Comment:
Is 9999 the maximum number allowed>

Reply:
Yes, that was the maximum number specified by the MR MHV devs for the API. We are trying to get as many records as possible per API call.